tests/parsers/test_dos/test_dos_default.yml

```diff
@@ -10,6 +10,6 @@ dos:
   - states/eV
   - states/eV
 parameters:
-  code_version: v.6.4.1
+  code_version: 6.4.1
```

Comment:
Should we keep this the same to not break backwards-compatibility?

Reply:
Hmm, perhaps. I preferred removing the `v.`, because it's cleaner and more consistent with e.g. the plugin version added to the metadata. But it's not a big deal, so probably backwards compatibility is the stronger argument.

Reply:
Looking at this again, I'd be less shy about breaking backwards compatibility regarding this version number. It's probably not being used at all atm, and removing the `v.` is cleaner in my view.